pkg/minikube/cni/cni.go

```diff
@@ -183,3 +197,33 @@ func applyManifest(cc config.ClusterConfig, r Runner, f assets.CopyableFile) err
 
 	return nil
 }
+
+// configureCNI - to avoid conflicting CNI configs, it sets:
+// - for crio: 'cni_default_network' config param via cni.Network
+// - for containerd and docker: kubelet's '--cni-conf-dir' flag to custom CNI Config Directory path (same used also by CNI Deployment).
+// ref: https://github.com/kubernetes/minikube/issues/10984 and https://github.com/kubernetes/minikube/pull/11106
+// Note: currently, this change affects only Kindnet CNI (and all multinodes using it), but it can be easily expanded to other/all CNIs if needed.
+// Note2: Cilium does not need workaround as they automatically restart pods after CNI is successfully deployed.
+func configureCNI(cc *config.ClusterConfig, cnm Manager) error {
+	if _, kindnet := cnm.(KindNet); kindnet {
```

Comment:
What about others ? Cillium,...

Reply:
Don’t ignore error, handle it or log it

Reply:
if i remember correctly, cilium works differently and does not need this - ie, automatically restarting the pods that depend on it (eg, coredns) right after cilium is deployed and initialised

it would be best to first update other CNIs and then see/test if this is needed

ok, i'll handle/log the error here